get-docker.sh

```diff
@@ -0,0 +1,476 @@
+#!/bin/sh
```

Comment:
why is this file in the repo? If there is a very specific reason, should it be at the top level?